### PR TITLE
Staff/IC Interactions chat setting.

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Adds a chat-tab setting for Staff-IC interactions such as Faxes and prayers.

# Explain why it's good for the game
Allows staff to filter things out so they can reply to faxes and similar easier.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Adds a staff to IC chat tab setting for faxes and prayers.
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
Uh wait stop no can't all Ghosts see Faxes ?

**Comment by realforest2001** [comment]
> Uh wait stop no can't all Ghosts see Faxes ?

Yes? It doesn't stop people seeing them. It's not a staff-only thing, it's just chat formatting. Also, anything that's in StaffIC is also in Info

File: code/game/machinery/fax_machine.dm [hunk 1]
```diff
@@ -342,7 +342,7 @@ var/list/alldepartments = list()
 			GLOB.GeneralFaxes.Add("<a href='?FaxView=\ref[faxcontents]'>\[view message at [world.timeofday]\]</a> <a href='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];CLFaxReply=\ref[user];originfax=\ref[src]'>REPLY</a>")
 			msg_admin += "(<a href='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];USCMFaxReply=\ref[user];originfax=\ref[src]'>RPLY</a>)</b>: "
 
-	msg_admin += "Receiving fax via secure connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>"
+	msg_admin += SPAN_ADMIN_IC("Receiving fax via secure connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>")
```
Comment (realforest2001) [inline code/game/machinery/fax_machine.dm:345]:
```suggestion
	msg_admin += SPAN_STAFF_IC("Receiving fax via secure connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>")
```

**Review by fira — APPROVED** [review]
Oh yeah OK both the ghost and admin messages are handled separately

Worth saying - Not sure how TMable this is, tgchat pulled some real funsies on us in the past with invalid message types (remember ERT)

**Pull request merged**